features/step_definitions/record_of_guidance_steps.rb

```diff
@@ -13,6 +15,8 @@
 
 Then(/^a record of guidance document is created$/) do
   expect(page.response_headers['Content-Type']).to eql('application/pdf')
-  text = PDF::Inspector::Text.analyze(page.source).strings.join
-  expect(text).to include('Joe Bloggs')
+end
+
+Then(/^emailed to the customer$/) do
+  expect(ActiveJob::Base.queue_adapter.enqueued_jobs.first[:args][0]).to eql('OutputDocumentMailer')
```

Comment:
You could introduce https://github.com/gocardless/rspec-activejob as a dependency?